enctests/testframework/main.py

```diff
@@ -655,31 +731,57 @@ def main():
         create_source_config_files(args)
         return
 
-    # Make sure we have a destination folder
-    Path(args.encoded_folder).mkdir(parents=True, exist_ok=True)
-
```

Comment:
I'm assuming this is taken car of elsewhere?

Reply:
Yes, its moved to the encoder-factory, encoders/__init__.py